src/platforms/stm32/src/lib/sys.c

```diff
@@ -17,20 +17,25 @@
  *
  * SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
  */
+#include <time.h>
 
 #include <avmpack.h>
 #include <defaultatoms.h>
+#include <interop.h>
```

Comment:
Maybe I'm missing something, do we need to add <interop.h> after this change?

Reply:
No. This was a mistake. I think it was accidentally included in a stash, and I missed it when I applied.

Reply:
removed.